plat/qemu/qemu_bl1_setup.c

```diff
@@ -57,9 +57,15 @@ void bl1_early_platform_setup(void)
  * does basic initialization. Later architectural setup (bl1_arch_setup())
  * does not do anything platform specific.
  *****************************************************************************/
+#if ARM_ARCH_MAJOR == 7
```

Comment:
Is it possible for QEMU to support AArch32 mode on ARMv8? In that case would prefer `#ifdef AARCH32` check here. Similarly other places if is the case.

Reply:
ok, I will use AARCH32 rather than ARM_ARCH_MAJOR when possible.